Purchases/Public/RCPurchases.h

```diff
@@ -375,6 +375,26 @@ NS_SWIFT_NAME(restoreTransactions(_:));
            withDiscount:(SKPaymentDiscount *)discount
         completionBlock:(RCPurchaseCompletedBlock)completion NS_SWIFT_NAME(purchasePackage(_:discount:_:)) API_AVAILABLE(ios(12.2), macosx(10.14.4));
 
+
+/**
+ Invalidates the cache for purchaser information.
+ This is useful for cases where purchaser information might have been updated outside of the app, like if a
+ promotional subscription is granted through the RevenueCat dashboard.
+ */
+- (void)invalidatePurchaserInfoCache;
+
+#pragma mark Subscriber Attributes
+
+- (void)setAttributes:(NSDictionary<NSString *, NSString *> *)attributes;
```

Comment:
We should add some docs here @aboedo

Reply:
good catch, updating

Reply:
https://github.com/RevenueCat/purchases-ios/pull/196/commits/d0f79328e11e01dc588d7e790579e807c57ca2de